File: app/src/main/java/com/example/maksim/testapp/details/presenter/DetailsModelListener.java
package com.example.maksim.testapp.details.presenter;

import com.example.maksim.testapp.details.model.data.GitHubUserDetails;

/**
 * Created by Maksim on 2017-09-12.
 */

public interface DetailsModelListener {
    void onResponse(GitHubUserDetails userDetails);
    void onFailureRequest(String message);
}

File: app/src/main/java/com/example/maksim/testapp/details/presenter/DetailsModelListener.kt
package com.example.maksim.testapp.details.presenter

import com.example.maksim.testapp.details.model.data.GitHubUserDetails

/**
 * Created by Maksim on 2017-09-12.
 */

interface DetailsModelListener {
    fun onResponse(userDetails: GitHubUserDetails)
    fun onFailureRequest(message: String)
}
